src/cms-content/vaccines/index.ts

```diff
@@ -35,15 +35,19 @@ export const useRegionTypeVaccineData = (regionType: RegionType) => {
   return data;
 };
 
-export const useRegionVaccineData = (region: Region) => {
-  const [data, setData] = useState<RegionVaccinationInfo>();
-  const regionTypeData = useRegionTypeVaccineData(region.regionType);
+export const useRegionVaccineData = (
+  regionType: RegionType,
+  regions: Region[],
```

Comment:
Can we determine what to fetch based on the `regions` argument? I'm not sure we really need to use the `regionType` argument here

Reply:
It's a bit tricky because of the way the hooks work right now.  I think we'll be able to clean this up a bit but can deal with it when we add other region types